firebase-perf/src/main/java/com/google/firebase/perf/internal/RemoteConfigManager.java

```diff
@@ -49,7 +53,7 @@
 
   private static final String FIREPERF_FRC_NAMESPACE_NAME = "fireperf";
 
-  private static final RemoteConfigManager sharedInstance = new RemoteConfigManager();
+  private static final RemoteConfigManager sharedInstance = DaggerRemoteConfigManagerComponent.create().getRemoteConfigManager();
```

Comment:
You cannot use `DaggerRemoteConfigManagerComponent.create().getRemoteConfigManager()` to get `RemoteConfigManager` when you are inside `RemoteConfigManager`. Instead, you have to let Dagger create and inject `RemoteConfigManager` to the class that calls `RemoteConfigManager.getInstance`. For the same reason, `RemoteConfigManagerComponent.java` may not be needed.